### PR TITLE
[@types/google-apps-script] Add DisplayStyle to CardService interface

### LINKED ISSUE #46898: [@types/google-apps-script] CardService is missing methods
Recently I've been trying to add a footer, if you try to do the suggested cat tutorial idea: https://developers.google.com/gsuite/add-ons/cats-quickstart, it cannot find `setFixedFooter()`

area of concern:
```
...

// Create a footer to be shown at the bottom.
  var footer = CardService.newFixedFooter()
      .setPrimaryButton(CardService.newTextButton()
          .setText('Powered by cataas.com')
          .setOpenLink(CardService.newOpenLink()
              .setUrl('https://cataas.com')));

  // Assemble the widgets and return the card.
  var section = CardService.newCardSection()
      .addWidget(image)
      .addWidget(buttonSet);
  var card = CardService.newCardBuilder()
      .addSection(section)
      .setDisplayStyle(DisplayStyle.REPLACE) // cannot find setDisplayStyle and DisplayStyle enum
      .setFixedFooter(footer); // cannot find
...

```

note: on version:     "@types/google-apps-script": "^1.0.14"

@PopGoesTheWza , @grant

### PR DESCRIPTION
**Pull request by davidbielik**
Fixes #46898

* Adds `DisplayStyle` to `CardService` interface ([docs](https://developers.google.com/apps-script/reference/card-service/card-builder#setDisplayStyle(DisplayStyle)))

```typescript
CardService.newCardBuilder()
  .setDisplayStyle(CardService.DisplayStyle.REPLACE)
```

image

--

- [ x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x ] Test the change in your own code. (Compile and run.)
- [ x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/google-apps-script/google-apps-script.card-service.d.ts [hunk 1]
```diff
@@ -252,6 +252,7 @@ declare namespace GoogleAppsScript {
       ComposedEmailType: typeof ComposedEmailType;
       ContentType: typeof ContentType;
       GridItemLayout: typeof GridItemLayout;
+      DisplayStyle: typeof DisplayStyle;
```
Comment (mtgto) [inline types/google-apps-script/google-apps-script.card-service.d.ts:255]: Sort alphabetical.
```suggestion
      DisplayStyle: typeof DisplayStyle;
      GridItemLayout: typeof GridItemLayout;
```

**Comment by mtgto** [comment]
Thanks! I miss #46898.

**Review by mtgto — APPROVED** [review]
👍

**Comment by davidbielik** [comment]
Ready to merge

**Pull request merged**